### DISCUSSION TITLE
Automatically standardize screen size

### DISCUSSION THREAD

**Discussion by lily2brain** [jspsych/jsPsych#1791] [Q&A] [answered]
Hi, I'm having trouble trying to make the images in my experiment look the same size, regardless of the size of my participant's computer screen. Is it possible to do that? I've tried here (https://www.jspsych.org/plugins/jspsych-resize/) but it didn't work. I await a help :)

**Comment by GEJ1** [comment, marked as answer]
Hi @lily2brain ,

From jspsych version v6.3.0 jspsych-resize plugin was merged with [jspsych-virtual-chinrest](https://www.jspsych.org/plugins/jspsych-virtual-chinrest/). Maybe that's the problem?

The idea behind it's the following:

> To calculate the pixel-to-cm conversion rate for a participant’s display, participants are asked to place a credit card or other item of the same size on the screen and resize an image until it is the same size as the credit card. Since we know the physical dimensions of the card, we can find the conversion rate for the participant's display.

So finally you can present a stimuli of the same absolute size in centimeters in different screens.

I don't know a better way to do that  in jspsych. Maybe you can share the piece of code that it's not woking and we can figure it out.

Best,

Gustavo.

**Reply by lily2brain** [reply]
Hi, it worked using jspsych-resize, there was no insert in the jsPsych.init timeline. Thanks :)
